auth/basic-auth/Dockerfile

```diff
@@ -1,25 +1,32 @@
-FROM teamserverless/license-check:0.3.6 as license-check
+FROM --platform=${BUILDPLATFORM:-linux/amd64} teamserverless/license-check:0.3.6 as license-check
```

Comment:
I would expect this to need to use 0.3.9 to support the newer multi-arch tag

Reply:
0.3.6 is also multi-arch. https://hub.docker.com/layers/teamserverless/license-check/0.3.6/images/sha256-02fe7269f4cf17fdc6c8a67395a8c7feb68d875c6a688a2134d19f96a02546c5?context=explore